src/Nethermind/Nethermind.Core.Test/Builders/BlockBuilder.cs

```diff
@@ -94,7 +94,8 @@ public BlockBuilder WithTransactions(int txCount, ISpecProvider specProvider)
             }
 
             long number = TestObjectInternal.Number;
-            ReceiptTrie receiptTrie = new(specProvider.GetSpec(number), receipts);
+            ulong timestamp = TestObjectInternal.Timestamp;
+            ReceiptTrie receiptTrie = new(specProvider.GetSpec(number, timestamp), receipts);
```

Comment:
Add extension method in tests that would take `TestObjectInternal` type?

Reply:
Do you really feel the need for that?